include/bluetooth/bluetooth.h

```diff
@@ -1290,6 +1290,22 @@ struct bt_le_per_adv_sync_param {
  */
 uint8_t bt_le_per_adv_sync_get_index(struct bt_le_per_adv_sync *per_adv_sync);
 
+/** @brief Get advertiser address address of a periodic adv sync object.
```

Comment:
address address

That would be the address of the remote, right?
I'm wondering if in the future there would be other information needed and a more generic bt_le_per_adv_sync_get_info function would be more suited to retrieve this address, this is also something that we have
already for connections and extended advertiser.

Reply:
Fixed the address address. 

The get info might be a good alternative, as there are a few other fields that we might want to expose at a later time. Should these coexist, or should the get info replace this?

Reply:
I prefer replacing, it is similar for connections, addresses are included in the info.